news/2021-04-11-osu-beatmapping-world-championship-2021-call-for-judges.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "osu! Beatmapping World Championship 2021: Call for Judges"
+date: 2021-04-11 14:00:00 +0000
+---
+
+With the Spring season comes the next iteration of osu! Beatmapping World Championship! If you're excited and eager to help us out, here's your chance: We are looking for judges!
```

Comment:
```suggestion
With the Spring season comes the next iteration of osu! Beatmapping World Championship! If you're excited and eager to help us out, here's your chance, because we're looking for judges!
```
nit